cfgov/unprocessed/css/organisms/mega-menu.less

```diff
@@ -4,132 +4,137 @@
+                                                                <li class="list_item
+                                                                           o-mega-menu_content-item">
+                                                                    <a class="o-mega-menu_content-link"
+                                                                       href="/the-bureau/">
+                                                                        The Bureau
+                                                                     </a>
+                                                                </li>
+                                                                …
+                                                            </ul>
+                                                            <ul class="list
+                                                                       list__unstyled
+                                                                       o-mega-menu_content-list">
+                                                                    <li class="list_item
+                                                                               o-mega-menu_content-item">
+                                                                        <a class="o-mega-menu_content-link__current
+                                                                                  o-mega-menu_content-link"
+                                                                           href="/careers/">
+                                                                            Careers
+                                                                         </a>
+                                                                    </li>
+                                                                    …
+                                                            </ul>
+                                                            …
+                                                        </div>
+                                                    </div>
+                                                    <aside class="sub-nav_media">
+                                                        [Featured Menu Content]
+                                                    </aside>
+                                                </div>
+                                            </div>
+                                        </div>
+                                    </li>
+                                </ul>
+                                [Global Eyebrow]
                             </div>
                         </div>
-                    </li>
-                </ul>
-                [Global Eyebrow]
-            </nav>
-        </div>
+                    </div>
+                </div>
+            </div>
+        </nav>
       codenotes:
         - |
           Structural cheat sheet:
           -----------------------
```

Comment:
Haha, it is a bit of a beast.
I find the structural cheat sheet still a bit useful since it collapses the dom down and makes it easier to grok the structure while building it. However, the markup example further up I feel less attached to. It's more dense and doesn't really offer much over just looking at the output in the page itself.